Code/Framework/AzFramework/Tests/ArchiveCompressionTests.cpp

```diff
@@ -35,6 +35,14 @@ namespace UnitTest
             : m_application { AZStd::make_unique<AzFramework::Application>() }
         {}
 
+        // Creates a unique filename for the archive to make the tests atomic and prevent file race conditions when
+		// tests are run concurrently
```

Comment:
This works for this scenario.

I was thinking a simpler approach with less changes would be to add a `AZ::Test::ScopedAutoTempDirectory m_tempDirectory;` member to this fixture.
Then in the constructor for the `ArchiveCompressionTestFixture`, set the `@usercache@` to that directory
```c++
AZ::IO::FileIOBase* fileIo = AZ::IO::FileIOBase::GetInstance();
fileIo->SetAlias("@usercache@", m_tempDirectory.GetDirectory());
```

Reply:
Great suggestion, fixed 👍